core/src/test/resources/integration_test_files/helloworld/get_api_integration_test.json

```diff
@@ -4,7 +4,7 @@
         {
             "name": "find_match",
             "url": "/api/v1/search/persons",
-            "method": "GET",
+            "method": "POST",
```

Comment:
`POST` >> Should be `GET`
Can you update it?
Please wait for the build status too...

Reply:
Fixed!